python/MyriaPythonWorker.py

```diff
@@ -123,6 +124,9 @@ def write_with_length(cls, obj, stream, output_type):
         elif output_type == DataType.BLOB:
             cls.write_int(DataType.BLOB, stream)
             cls.pickle_and_write(obj, stream)
+        elif output_type ==DataType.EXCEPTION:
+            cls.write_int(len(obj), stream)
```

Comment:
This is exceptions, not any python objects, typically a  string.  we convert this to a string with UTF_8 standard character set on the java side. 
I believe this should be fine.